app/components/article-featured-video.js

```diff
@@ -43,7 +46,7 @@ export default Component.extend(
 		 */
 		initOnScrollBehaviour() {
 			const $video = this.$('.article-featured-video__container'),
-				videoHeight = $video.height(),
+				videoHeight = this.get('withinPortableInfobox') ? 97 : 230,
```

Comment:
The video height isn't constant, it depends on the viewport width, we keep the aspect ratio.